workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/PurchasedAppsDashboardTableRow.tsx

```diff
@@ -138,6 +139,22 @@ export function PurchasedAppsDashboardTableRow({
 					}
 				>
 					<DropDown.ItemList>
+						{orderTypeExternalReferenceCode === orderType.DXP && (
+							<ClayTooltipProvider>
+								<DropDown.Item
+									data-tooltip-align="left"
+									disabled={orderStatusIsNotCompleted}
+									onClick={() => {}}
```

Comment:
Waiting for Keven's structural changes to be merged, to create the full onClick function

Reply:
Great joma!

Reply:
Waiting for: https://github.com/brianchandotcom/liferay-portal/pull/142463#event-10716511445